### PR TITLE
make otel process stacktraces into structured format

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

* Makes our otel errors ingest parse stack traces into a structured format that the frontend can render.
* Resolves a noisy log `PushMetricsImpl called without secureID` from session-less metrics (not yet supported).

## How did you test this change?

New unit test, full stack deploy.

## Are there any deployment considerations?

No.

### CODE REVIEW

File: backend/otel/parse_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	f.Fuzz(func(t *testing.T, stackTrace string) {
+		formatStructureStackTrace(stackTrace)
+	})
+}
+
+func TestFormatStructureStackTrace(t *testing.T) {
+	var inputs = []struct {
+		language           string
+		stacktrace         string
+		expectedFrameError string
```
Comment (et) [inline backend/otel/parse_test.go:18]: What are we using this for? Should this be something we're asserting on down below?

File: backend/otel/parse_test.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package otel
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func FuzzFormatStructureStackTrace(f *testing.F) {
+	f.Fuzz(func(t *testing.T, stackTrace string) {
```
Comment (et) [inline backend/otel/parse_test.go:9]: This is really cool!

File: backend/public-graph/graph/resolver.go [hunk 3]
```diff
@@ -2047,10 +2047,12 @@ func (r *Resolver) AddLegacyMetric(ctx context.Context, sessionID int, name stri
 
 func (r *Resolver) PushMetricsImpl(ctx context.Context, sessionSecureID string, metrics []*publicModel.MetricInput) error {
 	span, _ := tracer.StartSpanFromContext(ctx, "public-graph.PushMetricsImpl", tracer.ResourceName("go.push-metrics"))
+	span.SetTag("SessionSecureID", sessionSecureID)
+	span.SetTag("NumMetrics", len(metrics))
 	defer span.Finish()
 
 	if sessionSecureID == "" {
-		return e.New("PushMetricsImpl called without secureID")
+		return nil
```
Comment (et) [inline backend/public-graph/graph/resolver.go:2055]: Thanks for fixing this.

**Review by et — APPROVED** [review]
One q about testing but not blocking.

File: backend/otel/parse_test.go (hunk 1 above)
Comment (Vadman97) [reply backend/otel/parse_test.go:18]: whoops! accidently removed the `expectedFrameError` check while testing. good catch!

**Pull request merged**